flutter/cpp/mlperf_driver.cc

```diff
@@ -105,6 +105,7 @@ void MlperfDriver::RunMLPerfTest(const std::string& mode, int min_query_count,
   // See https://github.com/mlcommons/inference/issues/1397
   mlperf_settings.max_duration_ms =
       static_cast<uint64_t>(std::ceil(max_duration * 1000.0));
+  mlperf_settings.enforce_max_duration = false;
```

Comment:
Should we make this configurable? And where to set it?

Reply:
There are 2 flags:
* max_duration: control if the benchmark run should be stopped if it runs longer than max_duration.
* enforce_max_duration: should be named something like over_max_duration_result_valid because it controls if a result is still valid if it runs longer than max_duration.